build-logic/profiling/src/main/kotlin/gradlebuild/buildscan/tasks/ExtractCodeQualityBuildScanData.kt

```diff
@@ -101,7 +101,7 @@ abstract class ExtractCodeNarcBuildScanData : AbstractExtractCodeQualityBuildSca
                         getElementsByTag("Message").first()
                             ?: getElementsByTag("SourceLine").first()
                     }
-                    "$filePath:${violation.attr("lineNumber")} \u2192 ${message.text()}"
+                    "$filePath:${violation.attr("lineNumber")} \u2192 ${message?.text() ?: "Message not found"}"
```

Comment:
I just noticed this, why was this change needed?

Reply:
jsoup added `@Nullable` to the `Elements.first()`, so now `message` has type `Element?` instead of `Element!` (platform type) with the previous version.